application/electron/src/services/service.filters.ts

```diff
@@ -82,31 +79,7 @@ class ServiceFilters implements IService {
             }).catch((error: Error) => {
                 this._logger.warn(`Fail to subscribe to render event "FiltersSaveRequest" due error: ${error.message}. This is not blocked error, loading will be continued.`);
             });
-            ServiceElectron.IPC.subscribe(IPCMessages.FiltersFilesRecentRequest, this._ipc_onFiltersRecentRequested.bind(this)).then((subscription: Subscription) => {
-                this._subscriptions.FiltersFilesRecentRequest = subscription;
-            }).catch((error: Error) => {
-                this._logger.warn(`Fail to subscribe to render event "FiltersFilesRecentRequest" due error: ${error.message}. This is not blocked error, loading will be continued.`);
-            });
-            ServiceElectron.IPC.subscribe(IPCMessages.FiltersFilesRecentResetRequest, this._ipc_onFiltersFilesRecentResetRequested.bind(this)).then((subscription: Subscription) => {
-                this._subscriptions.FiltersFilesRecentResetRequest = subscription;
-            }).catch((error: Error) => {
-                this._logger.warn(`Fail to subscribe to render event "FiltersFilesRecentResetRequest" due error: ${error.message}. This is not blocked error, loading will be continued.`);
-            });
-            ServiceElectron.IPC.subscribe(IPCMessages.SearchRecentRequest, this._ipc_onSearchRecentRequest.bind(this)).then((subscription: Subscription) => {
-                this._subscriptions.SearchRecentRequest = subscription;
-            }).catch((error: Error) => {
-                this._logger.warn(`Fail to subscribe to render event "SearchRecentRequest" due error: ${error.message}. This is not blocked error, loading will be continued.`);
-            });
-            ServiceElectron.IPC.subscribe(IPCMessages.SearchRecentClearRequest, this._ipc_onSearchRecentClearRequest.bind(this)).then((subscription: Subscription) => {
-                this._subscriptions.SearchRecentClearRequest = subscription;
-            }).catch((error: Error) => {
-                this._logger.warn(`Fail to subscribe to render event "SearchRecentClearRequest" due error: ${error.message}. This is not blocked error, loading will be continued.`);
-            });
-            ServiceElectron.IPC.subscribe(IPCMessages.SearchRecentAddRequest, this._ipc_onSearchRecentAddRequest.bind(this)).then((subscription: Subscription) => {
-                this._subscriptions.SearchRecentAddRequest = subscription;
-            }).catch((error: Error) => {
-                this._logger.warn(`Fail to subscribe to render event "SearchRecentAddRequest" due error: ${error.message}. This is not blocked error, loading will be continued.`);
-            });
+            ServiceFileRecent.init();
```

Comment:
Should be included into the general queue of services `application/electron/src/main.ts`